examples/node/MarketDataSubscription_LongPoll.js

```diff
@@ -1,28 +1,35 @@
 var fs = require('fs');
 var util = require('util');
 var request = require("request");
+var cmdline = require("./cmdline");
 
-var host = process.argv[2] || 'http-api-host';
-var port = 3000;
-var url = 'https://' + host + ':' + port;
-var NUM_REQUEST = 10;   // Number of long-poll requests before unsubscribe
+var NUM_REQUEST = 10; // Number of long-poll requests before unsubscribe.
 var counter = 0;
-var opt = {
-    cert: fs.readFileSync('client.crt'),
-    key: fs.readFileSync('client.key'),
-    ca: fs.readFileSync('bloomberg.crt')
-};
+
+function url()
```

Comment:
Yeah when the modules get loaded in, the code within the modules will be executed sequentially. I guess it all depends on how you define your module. If we say it is a command line parsing utility, then having an explicit `parse` function definitely make more sense. I am ok with this way too, the only comment left would be the unnecessary of the `url` & `opt` functions. Personally I prefer the way you did it in the next example(`MarketDataSubscription_socketio.js`)